<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HierarchyDialog</class>
 <widget class="QWidget" name="HierarchyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hierarchy creation</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>75</width>
     <height>76</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="totalBool">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Total</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sup">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>41</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>41</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Supertype</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>371</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Entitys</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listEnt"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="pushAdd">
        <property name="text">
         <string>Add &gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushRemove">
        <property name="text">
         <string>&lt;&lt; Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Subtypes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listSub"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="exclusiveBool">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Exclusiva</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>9</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="modeComboBox"/>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>sup</zorder>
  <zorder>label</zorder>
  <zorder>buttonBox</zorder>
  <zorder>lineEdit</zorder>
  <zorder>line</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>totalBool</zorder>
  <zorder>exclusiveBool</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
